src/main.ts

```diff
@@ -143,3 +161,38 @@ function resolveVersionInput(): string {
 
   return version;
 }
+
+async function resolveStableVersionInput(
+  versionSpec: string,
+  auth: string | undefined,
+  arch = os.arch(),
+  releases: IToolRelease[]
+): Promise<string> {
+  if (versionSpec === StableReleaseAlias.Stable) {
+    core.info(`Stable version resolved as ${releases[0].version}`);
+
+    return releases[0].version;
+  } else {
+    const versions = releases.map(
+      release =>
+        `${semver.major(release.version)}.${semver.minor(release.version)}`
+    );
+    const uniqueVersions = Array.from(new Set(versions));
+
+    core.info(`Oldstable version resolved as ${uniqueVersions[1]}`);
```

Comment:
I think it is better to put core.info after `oldstableVersion` to show the full version.